src/renderer.h

```diff
@@ -250,6 +287,24 @@ class Renderer {
     uint16_t framebufferVertexOffsetX = 0;
     uint16_t framebufferVertexWidth = 0;
 
+    // Light Direction
+    struct point3d worldLightRot = { 60.0, 60.0, 0.0 };
+
+    // Shadowmap face culling
+    bool isShadowMapFaceCullingEnabled = true;
+
+    // Battle shadowmap frustum parameters
+    float shadowMapArea = 20000.0f;
+    float shadowMapNearFarSize = 20000.0f;
+
+    // Field shadowmap frustum parameters
+    float fieldShadowMapArea = 3000.0f;
+    float fieldShadowMapNearFarSize = 3000.0f;
+    
+    // Field Shadow walkmesh parameters
+    float walkMeshExtrudeSize = 20.0;
+    float walkMeshPosOffset = -11.0;
+

```

Comment:
As mentioned before, this should be part of the `internalState` structure, as the global variables here are only related to the `bgfx` class, whatever we do with it, and mutates ( get/set ) it must be in the `internalState`.

Reply:
I moved these variables to the internalState.